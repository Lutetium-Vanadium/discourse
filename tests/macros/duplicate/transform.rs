fn main() {
    let q = inquisition::questions![input {
        transform: todo!(),
        transform: todo!(),
    }];
}
