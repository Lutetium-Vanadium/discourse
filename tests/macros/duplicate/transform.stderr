error: duplicate option `transform`
 --> $DIR/transform.rs:4:9
  |
4 |         transform: todo!(),
  |         ^^^^^^^^^
